pkg/netpol/eval/resources.go

```diff
@@ -110,7 +113,7 @@ func (pe *PolicyEngine) resolveMissingNamespaces() error {
 // Deprecated: this function simply calls UpsertObject on the PolicyEngine.
 // Calling the UpsertObject should be preferred in new code.
 func (pe *PolicyEngine) SetResources(policies []*netv1.NetworkPolicy, pods []*corev1.Pod,
-	namespaces []*corev1.Namespace) error {
+	namespaces []*corev1.Namespace, services []*corev1.Service) error {
```

Comment:
this function is deprecated and only used in eval_test so I think we can currently avoid this change.